docs/tools/idx/quickstart.md

```diff
@@ -0,0 +1,99 @@
+# Quickstart
+
+Get started exploring what's possible with IDX using the [IDX SDK](../reference/idx.md). For usage in a Command Interface, you will need to [install the IDX CLI](../reference/cli.md).
+
+!!! warning ""
+
+    :octicons-alert-16: IDX is in alpha, Libraries may be unstable and APIs are subject to change. Data created on IDX during alpha will *not* be portable to production. Please share what you're working on and report any issues in the [IDX Discord](https://chat.idx.com){:target="_blank"}.
+
+!!! warning ""
+    :octicons-stop-16: If you haven't completed the Ceramic quickstart some information here may not make complete sense. We recommend coming here after you've completed the [Ceramic JavaScript Client Quickstart](../../build/javascript/quick-start.md).
+
+## **Prerequisites**
+
+The IDX SDK requires [Node.js](https://nodejs.org/){:target="_blank"} v14+ and npm v6+ (usually installed with Node.js). Make sure both are installed.
+
+On Linux you will also need the `libsecret` library to be installed, as [instructed here](https://github.com/atom/node-keytar#on-linux).
+
+## **Step 1: Install the IDX SDK**
+
+=== "Command"
+
+    ```bash
+    npm i @ceramicstudio/idx @ceramicnetwork/http-client
+    ```
+
+You will need to connect to a Ceramic node. The following is an example of connecting to a our Clay TestNet.
```

Comment:
```suggestion
You will need to connect to a Ceramic node. The following is an example of connecting to the Clay Testnet via the public gateway node hosted by 3BoxLabs.
```

Reply:
I don't think I've ever seen the "n" in Testnet capitalized before?  Not 100% confident on that though

Reply:
That might just be a me thing. Reasonable grammar fixes though.